packages/use-contractkit/src/modals/connect.tsx

```diff
@@ -53,10 +55,13 @@ export const ConnectModal: React.FC<ConnectModalProps> = ({
   const providers = useMemo<[providerKey: string, provider: Provider][]>(
     () =>
       Object.entries(PROVIDERS).filter(
-        ([, provider]) =>
+        ([providerKey, provider]) =>
           typeof window !== 'undefined' &&
           provider.showInList() &&
-          Object.keys(screens).find((screen) => screen === provider.name)
+          Object.keys(screens).find((screen) => screen === provider.name) &&
+          // eslint-disable-next-line
+          // @ts-ignore cant deduce that the keys are from the SupportedProviders enum
+          !denylist.includes(providerKey)
       ),
     [screens]
```

Comment:
Is there currently a way that denyList would ever be anything besides the default? If not I would just add an ignore for the 'missing dependency' lint warning